perllib/FixMyStreet/App/Controller/My.pm

```diff
@@ -234,6 +234,35 @@ sub by_shortlisted {
     }
 }
 
+sub anonymize : Path('anonymize') {
```

Comment:
It's also used to display the form, so I thought it didn't really matter (anyone malicious could try and construct a POST as easily as a GET, I mean), but yes it could check at the same point it checks the token. (Perhaps token check should always check POST, because you probably don't ever want a token in a GET…?)